OPi/GPIO.py

```diff
@@ -263,6 +264,23 @@ def _check_configured(channel, direction=None):
         raise RuntimeError("Channel {0} is configured for {1}".format(channel, descr))
 
 
+class PinDoesNotSupportEdgeDetection(Exception):
```

Comment:
Maybe rather than using a custom exception, just fail with a runtime exception like `_check_configured()` does ? 

Reason I ask is that because this is a programming error that needs a code level fix, not something that could/should be caught and the program continues.